.github/CODEOWNERS

```diff
@@ -7,6 +7,13 @@ src/snowflake/cli/api/utils/ @snowflakedb/snowcli @snowflakedb/nade
 **/test_data @snowflakedb/snowcli @snowflakedb/nade
 tests_e2e/conftest.py @snowflakedb/snowcli @snowflakedb/nade
 
+## Workspaces (shared ownership)
```

Comment:
we can make this more granular in the future, once all the pieces land in the right place, but I agree with this so far. Honour system as usual, we'll cross-review if anything spans more than a single team's area of responsibility.